dm/ui/src/App.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { lazy } from 'react'
+import { Provider } from 'react-redux'
+import { BrowserRouter, RouteObject, useRoutes } from 'react-router-dom'
+
+import { store } from '~/models'
+import Layout from '~/pages/_layout'
+
+const SourceList = lazy(() => import('~/pages/migration/source'))
+
+const routes: RouteObject[] = [
+  {
+    path: '/',
+    element: <Layout />,
+    children: [
+      {
+        index: true,
+        element: <div>TODO Dashboard</div>,
+      },
+      {
+        path: '/migration/task',
+        element: <div>TODO 任务列表</div>,
```

Comment:
i would like  all the Chinese to be translated into English (as this is an open-source project)